### DISCUSSION TITLE
Problemas no Publish do pacote MoneyExtension

### DISCUSSION THREAD

**Discussion by RafaelDifini** [balta-io/forum#78] [Tira Dúvidas] [answered]
CURSO: Fundamentos do Azure, Git, GitHub e DevOps

Segue como esta meu arquivo ..csproj, e o erro que tem retornado na action:

![image](https://user-images.githubusercontent.com/79666617/199785131-e61c4f7c-7aac-4c1a-8099-9e1ececcc5e8.png)

erro:
![image](https://user-images.githubusercontent.com/79666617/199785212-66e16d27-678c-436e-8e29-f47f9bf6860d.png)

a primeira vez que fiz o push, realmente a license estava como None, pois ja tinha tentao subir algo com MIT e tinha dado erro em outro repositorio, entao coloquei none no csproj pra testar, porem depois desse erro, eu alterei pra MIT, exclui o .nupkg, rodei o pack dnv, dei add, commit e push novamente, porem continuo com esse erro.

detalhe, tentei acessar o endpoint https://www.nuget.org/api/v2/package que é onde a action tenta publicar, e a pagina retorna 404

VOLTEI DUAS AULAS E VI ONDE ESTAVA O ERRO, desculpe xD
PackageLicenceExpression ainda estava como None

**Comment by andrebaltieri** [comment, marked as answer]
Perfeito!!! Que bom que deu certo @RafaelDifini
